cumulus/client/executor-gossip/src/lib.rs

```diff
@@ -88,27 +88,21 @@ impl Action {
 }
 
 /// Outcome of the network gossip message processing.
-#[derive(Debug)]
-pub enum HandlerOutcome<Error> {
-	/// The message is valid.
-	Good(Action),
-	/// The message is invalid.
-	Bad(Error),
-}
+pub type HandlerResult<E> = Result<Action, E>;
```

Comment:
Questionable usefulness to be honest, I'd prefer to see `Result<Action, Self::Error>`